Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -370,9 +368,7 @@ open class ChatChannelVC: _ViewController,
             }
         case is MarkUnreadActionItem:
             dismiss(animated: true) { [weak self] in
-                self?.channelController.markUnread(from: message.id) { _ in
-                    self?.updateUnreadMessagesRelatedComponents()
-                }
```

Comment:
@polqf This is not necessary. Whenever the reads are updated in the channel, the `didUpdateChannel` event is called so we can centralize updating the UI in this delegate event.